CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Changed
 
 - `Pagination`: always show arrows ([@qubis741](https://github.com/qubis741)) in [#2221](https://github.com/teamleadercrm/ui/pull/2221))
+- `Timer`: `timerRef` prop to `ref` ([@qubis741](https://github.com/qubis741)) in [#2221](https://github.com/teamleadercrm/ui/pull/2221))
```

Comment:
Great!! But isn't this a breaking change?

Reply:
I am not sure. @lorgan3 ?